scripts/agent/build-libbaihook.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
```

Comment:
Since we are also in charge of `backend.ai-hook` project it will be better to implement release pipeline of its own instead of just adding a single script in the dependent repository.